stl/inc/chrono

```diff
@@ -5226,12 +5226,12 @@ namespace chrono {
 
         // same as _Specs_setter
         constexpr void _On_fill(basic_string_view<_CharT> _Sv) {
-            if (_Sv.size() > 4) {
+            if (_Sv.size() > _STD size(_Specs._Fill)) {
                 _THROW(format_error("Invalid fill (too long)."));
             }
 
-            _STD fill(_Specs._Fill, _Specs._Fill + 4, _CharT{});
-            _STD copy(_Sv.begin(), _Sv.end(), _Specs._Fill);
+            auto _Pos = _STD _Copy_unchecked(_Sv._Unchecked_begin(), _Sv._Unchecked_end(), _Specs._Fill);
+            _STD fill(_STD move(_Pos), _STD end(_Specs._Fill), _CharT{});

```

Comment:
Moving `basic_string_view`'s unchecked iterator is unnecessary - we know that it doesn't benefit from move semantics.